content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+{{< glossary_definition term_id="eviction" length="all" >}}
+
+During an eviction, the kubelet sets the `PodPhase` for the selected pods to `Failed`.
+This gracefully terminates the pods. 
+
+If the pods are managed by a workload resource (such as {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}
+or {{< glossary_tooltip text="Deployment" term_id="deployment" >}}) that
+replaces failed pods, the control plane creates new pods in place of the evicted pods.
+
+{{<note>}}
+The kubelet attempts to [reclaim node-level resources](#reclaim-node-resources)
```

Comment:
might be helpful to give an example, like removing unused images when under disk pressure.